includes/functions.php

```diff
@@ -1447,7 +1447,7 @@ function rrdtest($path, &$stdOutput, &$stdError) {
 }
 
 function create_state_index($state_name) {
-    if (dbFetchRow('SELECT * FROM state_indexes WHERE state_name = ?', array($state_name)) === false) {
+    if (dbFetchRow('SELECT * FROM state_indexes WHERE state_name = ?', array($state_name)) == null) {
```

Comment:
As the state_name column can't be a null I'm not sure why we're checking against that. Would it make more sense to do !== true?